edgelet/edgelet-http-workload/src/server/mod.rs

```diff
@@ -34,7 +34,7 @@ pub struct WorkloadService {
 
 impl WorkloadService {
     // clippy bug: https://github.com/rust-lang-nursery/rust-clippy/issues/3220
```

Comment:
Can you check if this is fixed now?

Reply:
Looks like it - removed the allow and comment.